src/App.js

```diff
@@ -1,17 +1,16 @@
-import { Route, Routes } from 'react-router-dom';
-// import "./App.css";
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Books from './components/Books';
 import Categories from './components/Categories';
-import Header from './components/Header';
 
 function App() {
   return (
     <div className="App">
-      <Header />
-      <Routes>
-        <Route path="/" element={<Books />} />
-        <Route path="/categories" element={<Categories />} />
-      </Routes>
+      <Router>
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
+      </Router>
```

Comment:
- [ ] **[OPTIONAL]** We don't need to repeat this Header import in the Books and Categories components. In the App component, we can import it once.

  ```
  <Router>
    <Header />
    <Routes>
      <Route path="/" element={<Books />} />
      <Route path="/categories" element={<Categories />} />
    </Routes>
  </Router>
  ```